lib/Mojo/Pg/Migrations.pm

```diff
@@ -18,6 +18,20 @@ sub from_data {
   return $self->from_string(data_section($class //= caller, $name // $self->name));
 }
 
+sub from_dir {
+  my ($self, $dir) = @_;
+  path($dir)->list_tree({max_depth => 2})->map(
```

Comment:
This would probably be simpler with `each`.

Reply:
modified.